microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -101,7 +101,7 @@ The following table lists the currently available permission strings to use in y
 | `topSites` | Gives your extension access to the `chrome.topSites` API. |
 | `tts` | Gives your extension access to the `chrome.tts` API. |
 | `ttsEngine` | Gives your extension access to the `chrome.ttsEngine` API. |
-| `unlimitedStorage` | Provides an unlimited quota for storing client-side data, such as databases and local storage files.  Without this permission, the extension is limited to 5 MB of local storage. |
+| `unlimitedStorage` | Provides an unlimited quota for storing client-side data, such as databases and local storage files.  Without this permission, the extension is limited to 5 MB of local storage. ***Note**: This permission applies only to Web SQL Database and application cache (see issue 58985). Also, it doesn't currently work with wildcard subdomains such as http://*.example.com.*|
```

Comment:
```suggestion
| `unlimitedStorage` | Provides an unlimited quota for storing client-side data, such as databases and local storage files.  Without this permission, the extension is limited to 5 MB of local storage. ***Note**: This permission applies only to Web SQL Database and application cache (see issue 58985). Also, it doesn't currently work with wildcard subdomains such as http://*.contoso.com.*|
```
requirement: use contoso as placeholder
todo: link to Issue # , use accurate link text matching destination

Reply:
Is the trailing * correct?  ".com.*" seems odd.  "*.foo.com" seems expected.

Reply:
@JasonAndrewWriter - fyi see my Suggestion changes (dark green if you can see that above)